### PR TITLE
[OMI-1347] EasyForecast Modernisation

### PR DESCRIPTION
**Pull request by jacoboct**
Redone code and updating of SDK's to their latest versions.

### CODE REVIEW

File: EasyWeather/Advertising/AdNetworkConstants.swift [hunk 1]
```diff
@@ -28,15 +28,24 @@ let INTERSTITIAL_PLACEMENT = "interstitial_placement"
 let REWARDED_VIDEO_PLACEMENT = "rewarded_video_placement"
 
 //PubNative
+#if DEBUG
+let PUBNATIVE_APP_TOKEN = "543027b8e954474cbcd9a98481622a3b"
+let PUBNATIVE_BANNER_ZONE_ID = "2"
+let PUBNATIVE_MRECT_ZONE_ID = "3"
+let PUBNATIVE_INTERSTITIAL_ZONE_ID = "4"
+let PUBNATIVE_REWARDED_AD_ZONE_ID = "6"
+#else
```
Comment (can-soykarafakili) [inline EasyWeather/Advertising/AdNetworkConstants.swift:37]: you can clear the commented out ones @jacoboct

File: EasyWeather/Advertising/Unity/UnityInterstitialController.swift [hunk 2]
```diff
@@ -24,47 +24,90 @@
...
     
     override func loadAd() {
         adAnalyticsSession.start()
-        if (UnityAds.isReady(UNITY_INTERSTITIAL_AD_UNIT_ID)) {
+        UnityAds.load(UNITY_INTERSTITIAL_AD_UNIT_ID, loadDelegate: self)
+        if isUnityAdReady {
             unityAdsReady(UNITY_INTERSTITIAL_AD_UNIT_ID)
         } else {
-            unityAdsDidError(UnityAdsError(rawValue: 0)!, withMessage: "Error when loading the ad")
+            unityAdsDidError(UnityAdsInitializationError(rawValue: 0)!, withMessage: "Error when loading the ad")
```
Comment (can-soykarafakili) [inline EasyWeather/Advertising/Unity/UnityInterstitialController.swift:48]: Is `UnityAdsError` deprected @jacoboct ?

**Comment by jacoboct** [comment]
In this PR is not working ads from Facebook, Fyber and IronSource, this last one just show once banners and MRects but that changes are not included in this PR. Furthermore Fyber does not have implementation and I think Facebook ads are not working for a configuration problem in the dashboard.

**Review by faresbh — COMMENTED** [review]
@jacoboct build is still failing for me. 
`d: in /Users/faresbenhamouda/easy-forecast-ios/Pods/StartAppSDK/StartApp.framework/StartApp(STAMRAIDBridge.o), building for iOS Simulator, but linking in object file built for iOS, file '/Users/faresbenhamouda/easy-forecast-ios/Pods/StartAppSDK/StartApp.framework/StartApp' for architecture arm64`

**Review by can-soykarafakili — APPROVED** [review]
The app builds successfully with Rosetta 2 & overall looks okay. For any further updates we can create other tickets i believe 👍

**Comment by can-soykarafakili** [comment]
Also we can remove the commented lines as well. _(No need to create a separate ticket for it, we can do that inside another task @jacoboct )_

**Pull request merged**